packages/compass-components/src/components/generative-ai/generative-ai-input.spec.tsx

```diff
@@ -80,39 +81,62 @@ describe('GenerativeAIInput Component', function () {
   });
 
   describe('AIFeedback', function () {
-    it('should call the feedback handler on submit', async function () {
-      let feedbackChoice;
-      let feedbackText;
+    describe('when onSubmitFeedback is passed', function () {
+      let feedbackChoice: 'negative' | 'positive' | 'dismissed' | undefined;
+      let feedbackText: string | undefined;
+
+      beforeEach(function () {
+        feedbackChoice = undefined;
+        feedbackText = undefined;
+
+        renderGenerativeAIInput({
+          onSubmitFeedback: (_feedbackChoice, _feedbackText) => {
+            feedbackChoice = _feedbackChoice;
+
+            feedbackText = _feedbackText;
+          },
+          didSucceed: true,
+        });
+      });
 
-      renderGenerativeAIInput({
-        onSubmitFeedback: (_feedbackChoice, _feedbackText) => {
-          feedbackChoice = _feedbackChoice;
+      it('should have feedback options and call the feedback handler on submit', async function () {
+        // No feedback popover is shown yet.
+        expect(screen.queryByTestId(feedbackPopoverTextAreaId)).to.not.exist;
 
-          feedbackText = _feedbackText;
-        },
-        didSucceed: true,
-      });
+        const thumbsUpButton = screen.getByTestId(thumbsUpId);
+        expect(thumbsUpButton).to.be.visible;
+        thumbsUpButton.click();
+
+        const textArea = screen.getByTestId(feedbackPopoverTextAreaId);
+        expect(textArea).to.be.visible;
+        fireEvent.change(textArea, {
+          target: { value: 'this is the query I was looking for' },
+        });
```

Comment:
We usually use `userEvent` (and specifically `userEvent.type` here) for this

Reply:
Updated to use `userEvent` 👍 
Thanks for the callout, I'll update other uses in the repo as I come across them.